Documentation/ceph-filesystem-crd.md

```diff
@@ -94,6 +96,16 @@ spec:
     activeStandby: true
 ```
 
+**IMPORTANT**: For erasure coded pools, we have to create a replicated pool as the primary pool and erasureCoded as the secondary pool.
+Separately, a directory layout property needs to be added to use the secondary EC data pool.
+
+```console
+mkdir /mnt/cephfs/myssddir
```

Comment:
The directory will be created by the csi driver and mounted in the app pod. No need to create a new dir, right?